consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/converter/XmlToObjectTest.java

```diff
@@ -56,4 +63,73 @@ void shouldConvertBreathingSpaceRequestAndConvertToJson() throws IOException {
         assertNotNull(jsonString);
         assertEquals(JSON_RESPONSE, jsonString);
     }
+
+    @Test
+    void shouldConvertJudgementRequestToString() throws IOException {
+        String xmlContent = readXmlAsString(JUDGEMENT);
+        String request = xmlToObject.convertXmlToJson(xmlContent);
+        assertNotNull(request);
+        String expectedContent = readFile(JUDGEMENT_JSON);
+        assertEquals(expectedContent, request);
+    }
+
+    @Test
+    void shouldConvertJudgementRequest() throws IOException {
+        String xmlContent = readXmlAsString(JUDGEMENT);
+        JudgementRequest request = xmlToObject.convertXmlToObject(xmlContent, JudgementRequest.class);
+        assertNotNull(request);
+        assertEquals("9QZ00005", request.getCaseManRef());
+        assertEquals(JudgmentType.A, request.getJudgmentType());
+        assertEquals(true, request.isSentParticularsSeparately());
+        assertEquals(false, request.isJointJudgment());
```

Comment:
Could use assertTrue() and assertFalse() here instead of assertEquals().

Reply:
done